net-analyzer/ossec-hids/metadata.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>gentoo@seichter.de</email>
+		<name>Ralph Seichter</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<longdescription>
+		OSSEC is a full platform to monitor and control your systems. It mixes
+		together all the aspects of HIDS (host-based intrusion detection), log
+		monitoring and SIM/SIEM together in a simple, powerful and open source
+		solution.
+	</longdescription>
+	<use>
+		<flag name="agent">Build type "agent"</flag>
```

Comment:
Elaborating a bit here would be really helpful. Right now, the user hits 'you need to enable one of the options'. So he looks into the documentation and sees basically 'agent means agent'. Believe me, that's frustrating.

Reply:
I think that people who are interested in OSSEC would review the [installation](https://www.ossec.net/docs/manual/installation/install-source.html) and/or [architecture](https://www.ossec.net/docs/manual/ossec-architecture.html) docs, wouldn't you agree? OSSEC ist not something you just install and then see what it does. Would URLs in the flag descriptions be appropriate?

Reply:
Installation? Definitely not. The whole purpose of using a package manager is not to use upstream install scripts.

Reply:
Architecture, then. OSSEC is an intrusion detection system. One does not simply walk into Mor... I mean, one does not simply install it and hopes for the best. 😉 Reviewing the documentation is a must.

What about URLs in flag descriptions?

Reply:
Sure. However, I'd really still like the description to be somewhat… descriptory. Those names look rather well-defined, just the form of sentence makes them seem weird.

Reply:
How about adding something like this to the long description in `metadata.xml`:

> To determine which build target (agent, server, hybrid or local) suits your needs, please review the documentation provided via https://www.ossec.net/

If you have a better suggestion, please let me know.